modules/gdscript/tests/scripts/runtime/features/static_variables.out

```diff
@@ -5,7 +5,7 @@ data: data
 perm: 0
 prop: prefix Hello! suffix
 perm: 1
-prop: prefix World! suffix
+prop: <null>
```

Comment:
This is a bug. This is why I prefer not to use `--gdscript-generate-tests` but to edit `.out` manually, even if it's tedious.